src/style.css

```diff
@@ -1 +1,42 @@
 /* styles go here */
+.pesonal_information-input {
+  display: block;
+  padding-bottom: 10px;
+}
+
+.registration_input {
+  display: block;
+  padding-bottom: 10px;
+}
+
+.fact_input-color {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  display: block;
+}
+
+.fact_input {
+  display: block;
+  padding-bottom: 10px;
```

Comment:
no need to repeat all styles

you can add some common class to each elements or just use a comma
```
.fact_input,
.additional_input {
  display: block;
  padding-bottom: 10px;
}